rolisteam.json

```
@@ -0,0 +1,64 @@
+{
+    "id": "org.rolisteam.rolisteam",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "rolisteam",
+    "tags": ["1.9.0"],
+    "desktop-file-name-suffix": "",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--share=network", "--socket=pulseaudio", "--filesystem=home", "--env=TMPDIR=/var/tmp",
+    "--env=GST_PLUGIN_PATH=/usr/lib/x86_64-linux-gnu/gstreamer-1.0/" ],
+
+    "modules": [
+        {
+            "config-opts": [ "-DENABLE_TESTING=OFF" ],
+            "name": "libkdegames",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "git://anongit.kde.org/libkdegames.git" } ],
+            "modules": [
+        {
+            "config-opts": [ "-DENABLE_TESTING=OFF" ],
+            "name": "openal",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://github.com/kcat/openal-soft.git", "branch": "openal-soft-1.18.2" } ]
+        }
+          ]
+        },
+        {    "name": "rolisteam",
+           "buildsystem": "qmake",
+	   "post-install": [
+           "mv /app/rolisteam /app/bin/rolisteam",
+	   "rm /app/roliserver"
+	  ],
+           "sources": [
+                { "type": "git", "url": "https://invent.kde.org/kde/rolisteam.git" }
+            ]
+        },
+        {
+            "name": "rolisteam-data",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 org.rolisteam.rolisteam.desktop /app/share/applications/org.rolisteam.rolisteam.desktop",
+                "install -Dm644 org.rolisteam.rolisteam.appdata.xml /app/share/appdata/org.rolisteam.rolisteam.appdata.xml",
```

```suggestion
                "install -Dm644 org.rolisteam.rolisteam.metainfo.xml /app/share/appdata/org.rolisteam.rolisteam.metainfo.xml",
```
Rename the appdata file to metainfo.